Makefile

```diff
@@ -164,14 +180,19 @@ MACHINE_SIMPLE-$1=$$(MACHINE_SIMPLE-$$(ARCH-$1))
 else
 MACHINE_SIMPLE-$1=$$(ARCH-$1)
 endif
-SDK-$1=$$(basename $1)
+
+MACOSX_DEPLOYMENT_TARGET=10.6
```

Comment:
IMO there's not point supporting 10.6... IIRC current XCode only works with 10.8 onwards anyway.